### PR TITLE
Fixes #10827: replace usages of deprecated bst-infinite-scroll.

### PR DESCRIPTION
**Pull request by waldenraines**
http://projects.theforeman.org/issues/10827

### CODE REVIEW

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/errata/views/errata-table-full.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<table class="table table-striped table-bordered" ng-class="{'table-mask': table.working}">
+<table infinitclass="table table-striped table-bordered" ng-class="{'table-mask': table.working}">
```
Comment (ehelms) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/errata/views/errata-table-full.html]: Why is this the only instance that requires this change?

Comment (waldenraines) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/errata/views/errata-table-full.html]: That is a typo, will fix.

**Comment by ehelms** [comment]
[test]

File: rubygem-katello.spec [hunk 2]
```diff
@@ -87,8 +87,8 @@ Requires: postgresql
 Requires: postgresql-server
 Requires: v8
 Requires: %{?scl_prefix}rubygem-angular-rails-templates >= 0.0.4
-Requires: %{?scl_prefix}rubygem-bastion >= 1.0.0
-Requires: %{?scl_prefix}rubygem-bastion < 2.0.0
+Requires: %{?scl_prefix}rubygem-bastion >= 2.0.0
+Requires: %{?scl_prefix}rubygem-bastion < 3.0.0
```
Comment (ehelms) [inline rubygem-katello.spec]: Need to update the BuildRequires as well

Comment (waldenraines) [inline rubygem-katello.spec]: @ehelms updated.

**Comment by ehelms** [comment]
ACK

**Pull request merged**